include/class.search.php

```diff
@@ -454,7 +454,7 @@ static function createSearchTable() {
         $galera = db_result(db_query($sql));
 
         if ($galera && !$mysql56)
-            throw new Exception('Galera cannot be used with MyISAM tables');
+            throw new Exception('Galera cannot be used with MyISAM tables. Upgrade to MariaDB 10 / MySQL 5.6 is required');
```

Comment:
For mariadb 5.3 the support of InnoDB are enabled but the FULLTEXT indexes still not, so to have similar results, run manually REPAIR TABLE '<nametable>' QUICK over the iserach table WOITOUT DROPPING and solved.